pkg/rancher-desktop/components/Preferences/BodyVirtualMachine.vue

```diff
@@ -59,12 +59,18 @@ export default Vue.extend({
         },
       );
     },
+    selectTab(tab: string) {
+      const tabbed = this.$refs.tabbed as RdTabbed;
+
+      tabbed.$children[0].select(tab);
+    },
   },
 });
 </script>
 
 <template>
   <rd-tabbed
```

Comment:
`rd-tabbed` is just a wrapper for `Tabbed` and implements the same interface. This means that all of `Tabbed` props are exposed for us, so we can make use of the `activeTab` prop to navigate to a new tab via `:active-tab="activeTab"`.